firebase.agent.nut

```diff
@@ -594,16 +610,17 @@ class Firebase {
     // process the http response accordingly
     function _processResponse (request,callback) {
         request.sendasync(function(res) {
-            local data = res.body ;
+            local data = res.body;
             try {
-                data = http.jsondecode(data);
                 if (200 <= res.statuscode && res.statuscode < 300) {
-                    callback(null,data);
+                    data = http.jsondecode(data);
```

Comment:
I have never seen this. And I'm afraid of changing anything without a use case to verify.